### ISSUE TITLE
Update docs to use `migration_table` instead of `default_migration_table`.

### ISSUE THREAD

**Issue by MasterOdin** [cakephp/phinx#2332]
> On a side note, the docs still says `default_migration_table` but I think that's deprecated since `0.13`. 

 _Originally posted by @zkwbbr in [#2264](https://github.com/cakephp/phinx/issues/2264#issuecomment-1921995968)_

**Comment by IamMutevu** [comment]
Using ```0.14``` and it appears to still be using ```default_migration_table```. I guess the docs is still okay unless I am missing something

**Comment by MasterOdin** [comment]
Yeah, we never removed the functionality so it still works, but I plan to, but it'd be good to update the docs first to give people more lead time to do the right thing.

Granted, there's always a warning that gets printed when using `default_migration_table` to tell people to use `migration_table`.

**Comment by IamMutevu** [comment]
That's noted. Thanks for the heads up.

**Comment by dereuromark** [comment]
Long outdated I guess.

**Comment by MasterOdin** [comment]
Still something that needs updating in the docs, just haven't gotten to it yet unfortunately.
